swagger/coordinator.yml

```diff
@@ -0,0 +1,116 @@
+openapi: 3.0.1
+info:
+  title: XAIN FL Coordinator API
+  contact:
+    email: engineering@xain.io
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 1.0.0
+externalDocs:
+  description: Find out more about XAIN FL
+  url: https://docs.xain.io/
+servers:
+  - url: http://localhost:8081/
+tags:
+  - name: Coordinator
+    description: API of the coordinator service
+paths:
+  /rendez_vous:
+    get:
+      tags:
+        - Coordinator
+      responses:
+        200:
+          description: client accepted
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/RendezVousResponse"
+        404:
+          description: ""
+          content: {}
+  /heartbeat/{client_id}:
+    get:
+      tags:
+        - Coordinator
+      parameters:
+        - name: client_id
+          in: path
+          description: ID of the client
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: heartbeat confirmed
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/HeartBeatResponse"
+        404:
+          description: client unknown
+          content: {}
+  /start_training/{client_id}:
+    get:
+      tags:
+        - Coordinator
+      parameters:
+        - name: client_id
+          in: path
+          description: ID of the client
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: client is selected for training round
```

Comment:
The description should be something like "get the  information for fetching the latest global model"